submissions/Zakkarat/index.js

```diff
@@ -0,0 +1,92 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const INHABITANT = {
+    species: {
+        human: 'human'
+    },
+    genders: {
+        male: 'male',
+        female: 'female'
+    }
+};
+const Character = function (name, gender, legs, saying) {
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.saying = saying;
+  }
+  const Cat = function (name, gender, legs, saying) {
+    Character.call(this, name, gender, legs, saying);
+    this.species = "cat";
+  }
+  const Dog = function (name, gender, legs, saying) {
+    Character.call(this, name, gender, legs, saying);
+    this.species = "dog";
+  }
+  Character.prototype.sentence = function() {
+    let sentence = [
+        `Hey there, my name is <strong>${this.name}</strong>`,
+        `and I am a ${this.species}!`,
+        `My gender is ${this.gender},`,
+        `in addition I have ${this.legs} legs`,
+        `and ${this.hands} hands.`,
+        `Wait a minute... ${this.saying}`
+    ]
+    if(this.hands == null){
+      sentence.splice(4, 1);
+      sentence[3] = sentence[3].split("");
+      sentence[3].push(".");
+      sentence[3] = sentence[3].join("");
+    }
```

Comment:
Actually you should think about this block. There is much more easier way to do this. Something like work with you `hands` variable earlier...

Reply:
Done some changes. Could it be like this?

` sentence[4] = "."`